reference/conanfile/tools/google/bazeldeps.rst

```diff
@@ -0,0 +1,332 @@
+    # load_conan_dependencies()
+
+    def load_conan_dependencies():
+        native.new_local_repository(
+            name="zlib",
+            path="/path/to/conan/package/folder/",
+            build_file="/your/current/working/directory/zlib/BUILD.bazel",
+        )
+
+Given the example above, and imagining that your WORKSPACE is at the same directory, you would have to add these lines in there:
+
+.. code-block:: python
+    :caption: WORKSPACE
+
+    load("@//:dependencies.bzl", "load_conan_dependencies")
+    load_conan_dependencies()
+
+
+.. code-block:: python
+    :caption: zlib/BUILD.bazel
+
+    load("@rules_cc//cc:defs.bzl", "cc_import", "cc_library")
+
+    # Components precompiled libs
+    # Root package precompiled libs
+    cc_import(
+        name = "z_precompiled",
+        static_library = "lib/libz.a",
+    )
+
+    # Components libraries declaration
+    # Package library declaration
+    cc_library(
+        name = "zlib",
+        hdrs = glob([
+            "include/**",
+        ]),
+        includes = [
+            "include",
+        ],
+        visibility = ["//visibility:public"],
+        deps = [
+            ":z_precompiled",
+        ],
+    )
+
+    # Filegroup library declaration
+    filegroup(
+        name = "zlib_binaries",
+        srcs = glob([
+            "bin/**",
+        ]),
+        visibility = ["//visibility:public"],
+    )
+
+As you can observe, the *zlib/BUILD.bazel* defines these global targets:
+
+* ``zlib``: bazel library target. The label used to depend on it would be ``@zlib//:zlib``.
+* ``zlib_binaries``: bazel filegroup target. The label used to depend on it would be ``@zlib//:zlib_binaries``.
+
+Since Conan 1.62.0, you can put all the files generated by ``BazelDeps`` into another folder using the ``bazel_layout``:
+
+.. code-block:: python
+    :caption: conanfile.py
+
+    from conan import ConanFile
+    from conan.tools.google import BazelDeps, bazel_layout
+
+    class App(ConanFile):
+        settings = "os", "arch", "compiler", "build_type"
+        requires = "zlib/1.2.11"
+
+        def layout(self):
+            # DEPRECATED: Default generators folder will be "conan" in Conan 2.x
+            self.folders.generators = "conan"
+            bazel_layout(self)
+
+        def generate(self):
+            bz = BazelDeps(self)
+            bz.generate()
+
+
+Running again the :command:`conan install` command, we now get this structure:
+
+.. code-block:: bash
+
+    $ conan install .
+    .
+    ├── conan
+    │   ├── BUILD.bazel
+    │   ├── dependencies.bzl
+    │   └── zlib
+    │       └── BUILD.bazel
+    └── conanfile.py
+
+
+Now your Conan-bazel files were generated in the *conan/* folder, so your WORKSPACE will look like:
+
+.. code-block:: python
+    :caption: WORKSPACE
+
+    load("@//conan:dependencies.bzl", "load_conan_dependencies")
+    load_conan_dependencies()
+
+
+
+Build requirements
+++++++++++++++++++
```

Comment:
Being this a reference, I think that the title should have the attribute name, that is the entity you are documenting here
```suggestion
Build build_context_activated
+++++++++++++++++++++++++++++
```